.github/workflows/auto-assign-issues.yml

```diff
@@ -1,18 +0,0 @@
-name: Auto assign axe-core integration issues to Axe API team project board
-
-on:
-  issues:
-    types: [opened]
-env:
-  MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
```

Comment:
My guess is the reason this doesn't work is the token doesn't have sufficient permissions/scopes.

Also, I recommend using https://github.com/actions/add-to-project instead. It was authored by GitHub, so it probably works.